features/networking/sriov.feature

```diff
@@ -403,14 +403,13 @@ Feature: Sriov related scenarios
     When I execute on the pod:
       | bash | -c | cat /sys/class/net/net1/address |
     Then the step should succeed
-    And evaluation of `@result[:response].strip` is stored in the :pod1_net1_mac clipboard
+    And evaluation of `@result[:response].match(/\h+:\h+:\h+:\h+:\h+:\h+/)[0].strip` is stored in the :pod1_net1_mac clipboard
```

Comment:
`:response` contains both `:stdout` and `:stderr`.
You can use `:stdout` to get rid of `:stderr` msg.

Reply:
Thanks, updated.